2.segmentation/README.md

```diff
@@ -0,0 +1,9 @@
+# Image segementation of objects
+
+We need to segment the images into objects.
+We will use the Segement Anything Model (SAM) to do this.
```

Comment:
Might recommend including a link to SAM as a citation.

Reply:
added